index.js

```diff
@@ -0,0 +1,322 @@
+const keyboard_ru = [
+  { data: 192, value: '`' },
+  { data: 49, value: '1' },
+  { data: 50, value: '2' },
+  { data: 51, value: '3' },
+  { data: 52, value: '4' },
+  { data: 53, value: '5' },
+  { data: 54, value: '6' },
+  { data: 55, value: '7' },
+  { data: 56, value: '8' },
+  { data: 57, value: '9' },
+  { data: 48, value: '0' },
+  { data: 189, value: '-' },
+  { data: 187, value: '=' },
+  { data: 8, value: 'Backspace' },
+  { data: 9, value: 'Tab' },
+  { data: 81, value: 'й' },
+  { data: 87, value: 'ц' },
+  { data: 69, value: 'у' },
+  { data: 82, value: 'к' },
+  { data: 84, value: 'е' },
+  { data: 89, value: 'н' },
+  { data: 85, value: 'г' },
+  { data: 73, value: 'ш' },
+  { data: 79, value: 'щ' },
+  { data: 80, value: 'з' },
+  { data: 219, value: 'х' },
+  { data: 221, value: 'ъ' },
+  { data: 220, value: '' },
+  { data: 20, value: 'CapsLK' },
+  { data: 65, value: 'ф' },
+  { data: 83, value: 'ы' },
+  { data: 68, value: 'в' },
+  { data: 70, value: 'а' },
+  { data: 71, value: 'п' },
+  { data: 72, value: 'р' },
+  { data: 74, value: 'о' },
+  { data: 75, value: 'л' },
+  { data: 76, value: 'д' },
+  { data: 186, value: 'ж' },
+  { data: 222, value: 'э' },
+  { data: 38, value: '🠕' },
+  { data: 13, value: 'Enter' },
+  { data: 16, value: 'Shift' },
+  { data: 90, value: 'я' },
+  { data: 88, value: 'ч' },
+  { data: 67, value: 'с' },
+  { data: 86, value: 'м' },
+  { data: 66, value: 'и' },
+  { data: 78, value: 'т' },
+  { data: 77, value: 'ь' },
+  { data: 188, value: 'б' },
+  { data: 190, value: 'ю' },
+  { data: 191, value: '.' },
+  { data: 37, value: '🠔' },
+  { data: 40, value: '🠗' },
+  { data: 39, value: '🠖' },
+  { data: 32, value: 'Space' },
+  { data: 17, value: 'Ctrl' },
+  { data: 18, value: 'Alt' },
+  { data: 91, value: 'Win' },
+];
+
+if (localStorage.getItem('lang') === null) {
+  localStorage.setItem('lang', 'keyboard_en');
+}
+if (localStorage.getItem('CapsLK') === null) {
+  localStorage.setItem('CapsLK', 'false');
+}
+
+const keyboardFromLoc = localStorage.getItem('lang');
+const CapsLK = localStorage.getItem('CapsLK');
+
+const input = document.createElement('textarea');
+input.className = 'textarea';
+input.setAttribute('value', '');
+document.body.append(input);
+
+const div = document.createElement('div');
+div.className = 'keyboard_inner no-active';
+document.body.append(div);
+
+const p = document.createElement('p');
+p.className = 'keyboard_subTitel';
+document.body.append(p);
+p.innerHTML = 'Клавиатура создана на Windows <br> Нажмите ALT для переключения языка';
+
+if (keyboardFromLoc === 'keyboard_en') {
+  keyboard = keyboard_en;
+} else {
+  keyboard = keyboard_ru;
+}
+
+init(keyboard, CapsLK);
+
+function init(keyboard, CapsLK) {
+  let out = '';
+  if (CapsLK === 'true') {
+    for (let i = 0; i < keyboard.length; i++) {
+      out += `<div class="key" data="${keyboard[i].data}" value="${keyboard[
+        i
+      ].value.toUpperCase()}"> ${keyboard[i].value.toUpperCase()} </div>`;
```

Comment:
I think a better way to prepare value before concatenating result. It will be more readable.